packages/extension-ui/src/Popup/Signing/Request/SignArea.tsx

```diff
@@ -92,20 +94,14 @@ function SignArea({ buttonText, className, error, isExternal, isFirst, isLast, s
 
   const StyledCheckbox = styled(Checkbox)`
     margin-left: 8px;
-`;
+  `;
 
   const RememberPasswordCheckbox = () => (
     <StyledCheckbox
       checked={savePass}
-      label={
-        isLocked
-          ? t<string>('Remember password for {{expiration}} minutes', {
-              replace: { expiration: PASSWORD_EXPIRY_MIN }
-            })
-          : t<string>('Extend the period without password by {{expiration}} minutes', {
-              replace: { expiration: PASSWORD_EXPIRY_MIN }
-            })
-      }
+      label={t<string>('Remember password for {{expiration}} minutes', {
```

Comment:
I think it's great. Do we want to run it by Bryk / Toni?

Reply:
sure

Reply:
Waiting for Bryk's reply

Reply:
Bryk asked to do it that way: 
After password being saved for 15 minutes, user cant interact with the checkbox, it becomes disabled for this time, so user cant reset timer or clear cache manually by unchecking it as example, after 15 minutes checkbox enables again.